setup.py

```diff
@@ -189,9 +189,12 @@ def write_version(filename=os.path.join(*['airflow',
     'PyOpenSSL',
     'pandas-gbq'
 ]
-github_enterprise = ['Flask-OAuthlib>=0.9.1']
 grpc = ['grpcio>=1.15.0']
-google_auth = ['Flask-OAuthlib>=0.9.1']
+flask_oauth = [
+    'Flask-OAuthlib>=0.9.1',
+    'oauthlib!=2.0.3,!=2.0.4,!=2.0.5,<3.0.0,>=1.1.2',
```

Comment:
My that's quite the version specificiation! Where did it come from?

Reply:
I've based it on the master branch of the Flask-OAuthlib git repository:

https://github.com/lepture/flask-oauthlib/blob/9e6f152a5bb360e7496210da21561c3e6d41b0e1/setup.py#L44-L47

There were 2 commits to pin these versions:

https://github.com/lepture/flask-oauthlib/commit/bd06c64fec62e4f37aaed769d4f5156b30405e26
https://github.com/lepture/flask-oauthlib/commit/1379c6b27c17e3aac5ea071f12aca59f02c924b6

Similar to https://github.com/apache/airflow/pull/5036, we can unpin these once a new release of Flask-OAuthlib is released which includes the above changes.